assets/general/adamant/info.json

```
@@ -24,6 +24,10 @@
     "oldPendingInterval": 4000,
     "registeredInterval": 4000
   },
+  "messages": {
+    "messageTimeout": 300000,
+    "filesTimeout": 300000
+  },
```

I suggest this structure:

```suggestion
  "timeout": {
    "message": 300000,
    "file": 300000
  },
```

---

what about `file` -> `attachment`?

---

@yoxira also good

@adamant-al My only concern is about `messages` naming. It seems like we want to describe an array of something, which is obviously not true.

---

Accepted.
@bludnic Push the changes.